src/ios.ts

```diff
@@ -393,21 +393,46 @@ Components:" > /etc/apt/sources.list.d/appstraction.sources`,
         throw new Error('Failed to get prefs.');
     },
     async getDeviceAttribute(attribute, ...args) {
-        if (!options.capabilities.includes('frida'))
-            throw new Error('Frida is required for getting device attributes.');
+        // IDFV
+        if (attribute === 'idfv') {
+            if (!options.capabilities.includes('frida')) throw new Error('Frida is required for getting the IDFV.');
 
-        const opts = args[0]!;
+            const opts = args[0]!;
 
-        switch (attribute) {
-            case 'idfv': {
-                const pid = await this.getPidForAppId(opts.appId);
-                const idfv = await getObjFromFridaScript(pid, fridaScripts.getIdfv);
-                if (typeof idfv === 'string') return idfv;
-                throw new Error('Failed to get IDFV.');
-            }
+            const pid = await this.getPidForAppId(opts.appId);
+            const idfv = await getObjFromFridaScript(pid, fridaScripts.getIdfv);
+            if (typeof idfv === 'string') return idfv;
+            throw new Error('Failed to get IDFV.');
         }
 
-        throw new Error(`Unsupported device attribute: ${attribute}`);
+        // Manufacturer
+        if (attribute === 'manufacturer') return 'Apple';
+
+        // Attributes returned by `pymobiledevice3 lockdown info`
+        const lockdownAttributes = {
+            architectures: 'CPUArchitecture',
+            modelCodeName: 'HardwareModel',
```

Comment:
I don't know which one would be better suited. I think I went with `HardwareModel` because ["theappwiki.com"](https://theapplewiki.com/wiki/IPhone_X) (which I've seen quite a lot when looking up Apple stuff like this, so I consider them semi-authoritative :D) calls the `iPhone*,*` thingy[^mine] "firmware identifier", whereas it calls the other thing "model".

[^mine]: My iPhone X is `iPhone10,3` by the way.

Reply:
Ok, I misunderstood you. You mean _additionally_ for `model`, instead of replacing `modelCodeName`. Sorry.

Um… I guess… I feel like they are both fairly codename-y. Ideally, we'd have `model: iPhone X` here, but I definitely don't want to maintain (or import) a mapping just for that).

Reply:
Well, do you think we should just not have the `model` property? I think it would make sense, at least to differentiate between iPhone or iPad quickly.

Reply:
Iirc my thinking for not having `model` for iOS was: The "correct" information to put there would be the "human-readable"/marketing model name (e.g. iPhone X), but we don't (currently) have access to this. If we put something else there now, we effectively commit to that and even if we do gain the ability to retrieve the "correct" piece of information later, we are stuck with the "wrong" `model`.

Reply:
I think the firmware identifier or whatever is the best we are going to get. You can at least kind of tell which hardware was used and roughly how old it is by the number.

Reply:
Alternatively, we could also use `DeviceClass`, which is `iPhone` on my device.

Reply:
> I think the firmware identifier or whatever is the best we are going to get.

Why? If we actually _needed_ this information, we could always just import a database (from a _very_ quick glance: e.g. https://www.npmjs.com/package/idevicedb).

To me, it just seems wrong to have `model: moto g(7) power` on Android and `model: iPhone10,3` on iOS. Those are both clearly different types of information.

Reply:
> Alternatively, we could also use `DeviceClass`, which is `iPhone` on my device.

But that's different information yet again. If you think that `DeviceClass` and/or `ProductType` are useful, add additional properties instead of trying to conflate them into something they aren't. :D